lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -49,6 +53,10 @@
   white-space: nowrap;
 }
 
+@mixin monospace() {
```

Comment:
This is now redundant with the other mixin

Reply:
> This is now redundant with the other mixin

I kept `@monospace` because the naming was similar to our existing commonly used font mixins: https://github.com/sonatype/sonatype-react-shared-components/pull/273/commits/b92cf1cdf46a95a9a43a1ed5d5e2e5ed2af4cd36